SPECS/ding-libs/ding-libs.spec

```diff
@@ -9,38 +9,39 @@
 %global ini_config_version 1.3.1
 Summary:        "Ding is not GLib" assorted utility libraries
 Name:           ding-libs
-Version:        0.6.1
-Release:        45%{?dist}
+Version:        0.6.2
+Release:        55%{?dist}
```

Comment:
Can we reset this back to 1?

Reply:
I think this way I prevent less changes to the changelog, thoughts otherwise I dont know how to reconcile going in the change log from ver rel55 to rel 1 again

Reply:
Our general practice is to reset to 1 when the version gets bumped, i.e., from 0.6.1-45 to 0.6.2-1. The version bump disambiguates appropriately. If you look at some specs that have bumped versions more frequently you'll see some good examples in the change log...

Reply:
Okay, I understand now and stand corrected! I completely see why we should stick with `55` -- otherwise the Fedora changelog entries don't make sense...